docs/integrations/integration-basics/supported-transformations.md

```diff
@@ -0,0 +1,125 @@
+- [Event Types](../../cep/query-guide/event-types)
+- [Functions](../../cep/query-guide/functions/)
+- [Filters](../../cep/query-guide/filters/)
+- [Aggregate Functions](../../cep/query-guide/aggregate-functions)
+- [HAVING|WHERE](../../cep/query-guide/having-where)
+- [GROUP BY](../../cep/query-guide/group-by)
+- [ORDER BY](../../cep/query-guide/order-by)
+- [LIMIT and OFFSET](../../cep/query-guide/limit-and-offset)
+- [Output Rate Limiting](../../cep/query-guide/output-rate-limiting)
+- [In-line Windows](../../cep/windows/windows-queries), which are windows defined in the query.
+
+## Unsupported Stream QL
+
+The following Stream QL and stream worker elements are not supported. If you need to use them in your data pipeline, then you might need to create a stream worker:
+
+- [Custom Script Functions](../../cep/query-guide/custom-script-functions)
+- [JOIN](../../cep/query-guide/join/)
+- [Partitions](../../cep/query-guide/partition/)
+- [Named Windows](../../cep/windows/)
+- [Named Aggregations](../../cep/aggregations/)
+- [Ad Hoc Queries](../../cep/ad-hoc-queries/)
+- Creating any other elements, including [Tables](../../cep/table/), [Sources](../../cep/source/), and [SINKS](../../cep/sink/).
+
+## Syntax
+
+```sql
+SELECT
+    < expressions >
+FROM
+    _input
+WHERE
+    < condition > -- optional
+GROUP BY
+    < fields >
+HAVING
+    < condition > -- optional
+```
+
+## Example 1
+
+```sql
+INSERT INTO Output
+SELECT "Coinbase Pro" as exchange, 
+       "USA" as quote_region,
+        "BTC/USD" as symbol, 
+        avg(convert(price, 'double')) as ma, 
+        convert(price, 'double') as close,
+        time:timestampInMilliseconds()/1000 as timestamp
+FROM  Input;
+```
+
+This transformation calculates the average price, converts the price to a double data type, and adjusts the timestamp for a specific input.
+
+1. `"Coinbase Pro" as exchange, "USA" as quote_region, "BTC/USD" as symbol`: These sections are hardcoding the values for `exchange`, `quote_region`, and `symbol` fields to "Coinbase Pro", "USA", and "BTC/USD", respectively.
+
+2. `avg(convert(price, 'double')) as ma`: This takes the average of the `price` field, which is first converted to a `double` data type. The result will be aliased as `ma`.
+
+3. `convert(price, 'double') as close`: Here, the `price` is simply converted to a `double` data type, and the result is aliased as `close`.
+
+4. `time:timestampInMilliseconds()/1000 as timestamp`: The timestamp is converted from milliseconds to seconds and then renamed as `timestamp`.
+
+This transformation runs on all the documents from your `Input` collection and inserts the transformed documents into the `Output` collection.
+
+## Example 2
+
+```sql
+INSERT INTO Output
+SELECT exchange, 
+       quote_region, 
+       symbol, 
+       timestamp, 
+       trade_location,
+	     trade_price, 
+       trade_strategy, 
+       trade_type
+FROM Input
+WHERE
+	symbol == "MBDX" AND
+  trade_location = "CA";
```

Comment:
```suggestion
  trade_location == "CA";
```

Reply:
You might want to update this in the Notion doc if it is wrong, I copied it straight from there.